README.md

```diff
@@ -1,18 +1,40 @@
-# This project is in the early prototype / dreaming phase
-
 Hanoverd - The docker handover daemon
 -------------------------------------
 
-Hanoverd ("Hanover-Dee") is responsible for managing seamless transitions from
-one application version to another with Docker containers.
+Hanoverd ("Hanover-Dee") is responsible for managing seamless transitions
+from one application version to another with Docker containers.
 
-## Installation
+It's the [`docker replace`](https://github.com/docker/docker/issues/2733#issuecomment-123502548) command you always wanted.
+
+Status: beta. Things may change. We may need your feedback to make it work for you.
+
+## What is hanoverd good for?
+
+Let's say you have your web application running in docker, and you
+want to upgrade it seemlessly.
+
+You want to move fast and break things with continuous deployment,
+making updates to your app, but you don't want your web application
+to go down while it updates; or if it fails to restart.
+
+Hanoverd solves the problem by leaving the first working version
+running until a second one has started and correctly accepted a
+request.
 
-Hanoverd currently requires the ability to mess with iptables. This can be
-achieved with setcap to avoid using root.
+Hanoverd is ideal for stateless web applications running in docker
+where you can easily have multiple instances running side-by-side.
 
-A suitably capable `iptables` command must be on the `$PATH`. This can be
-achieved with this command:
+Hanoverd can pull or build images from a variety of sources and
+also listen to webhooks from github and elsewhere using an outbound
+TCP connection, rather than having to expose a port for listening to
+such events. This is done via [hookbot](https://github.com/scraperwiki/hookbot).
```

Comment:
I'd consider making the hookbot link in this line a link to the section mentioning hookbot, then add a link to hookbot in that section.

The hookbot section doesn't actually link to hookbot; a reader skimming past the last line of this paragraph may get to the bottom and be confused as to what hookbot is, or where they can find it.